src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -136,6 +136,32 @@
     }
 
     /**
+     * This method determines if a check should process javadoc containing unclosed html tags.
+     * This method must be overridden in checks extending {@code AbstractJavadocCheck} which
+     * are not supposed to process javadoc containing unclosed html tags.
+     * @return true if the check should or can process javadoc containing unclosed html tags;
+     *     false otherwise
+     * @see JavadocUtils#getNonTightHtmlTag(DetailNode) (DetailNode)
+     */
+    public boolean doesCheckAcceptJavadocWithNonTightHtml() {
+        return true;
+    }
+
+    /**
+     * This method determines if a check should log a violation upon encountering javadoc with
+     * non tight html. The default return value for this method is set to false since checks
+     * generally tend to be fine with non tight html.
+     * @return true if check is supposed to log a violation when a javadoc with non tight HTML
+     *     is encountered; false otherwise
+     * @see <a href="http://checkstyle.sourceforge.net/writingjavadocchecks.html#Tight-HTML_rules">
+     *     Tight HTML rules</a>
+     *
+     */
+    public boolean doesCheckViolateJavadocWithNonTightHtml() {
+        return false;
```

Comment:
@ps-sp I am talking about overriding the value of the property in the configuration, not the code.
Any field that has a setter in a module is a property in the configuration that can set, like tokens, etc...